pokemongo_bot/__init__.py

```diff
@@ -159,17 +159,13 @@ def work_on_cell(self, cell, position, work_on_forts=1):
         # Check if session token has expired
         self.check_session(position)
 
-        if self.config.initial_transfer:
```

Comment:
This flag was already inside of InitialTransferWorker so it was being checked twice.